using/unix.po

```diff
@@ -114,28 +130,40 @@ msgid ""
 "devguide.python.org/setup/#getting-the-source-code>`_.  (If you want to "
 "contribute patches, you will need a clone.)"
 msgstr ""
+"Si vous voulez compiler CPython vous-même, la première chose à faire est de "
+"récupérer le `code source <https://www.python.org/downloads/source/>`_. Vous "
+"pouvez télécharger la dernière version ou faire un `clone <https://devguide."
+"python.org/setup/#getting-the-source-code>`_. (Si vous voulez contribuer à "
+"des correctifs, il vous faut un clone.)"
 
 #: ../Doc/using/unix.rst:75
+#, fuzzy
 msgid "The build process consists in the usual ::"
-msgstr ""
+msgstr "Le processus de compilation est le suivant ::"
 
 #: ../Doc/using/unix.rst:81
 msgid ""
 "invocations. Configuration options and caveats for specific Unix platforms "
 "are extensively documented in the :source:`README.rst` file in the root of "
 "the Python source tree."
 msgstr ""
+"invocations. Les options de configuration et mises en garde pour des "
```

Comment:
```suggestion
"Les options de configuration et mises en garde pour des "
```

Le invocations est la fin de "The build process consists in the usual" mais tu l'as traduit entier.